test/built-ins/Array/prototype/find/predicate-call-parameters.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.8
+description: >
+  Predicate called as F.call( thisArg, kValue, k, O ) for each array entry.
+info: >
+  22.1.3.8 Array.prototype.find ( predicate[ , thisArg ] )
+
+  ...
+  6. If thisArg was supplied, let T be thisArg; else let T be undefined.
+  7. Let k be 0.
+  8. Repeat, while k < len
+    ...
+    d. Let testResult be ToBoolean(Call(predicate, T, «kValue, k, O»)).
+    e. ReturnIfAbrupt(testResult).
+  ...
+---*/
+
+var arr = ['Mike', 'Rick', 'Leo'];
+
+var results = [];
+
+arr.find(function(kValue, k, O) {
+  results.push({
```

Comment:
If you push the `arguments` object (instead of a collection of the expected formal parameters), you can also assert that the predicate wasn't incorrectly invoked with additional arguments (via `results[0].length`).